.github/workflows/license_review.yml

```diff
@@ -0,0 +1,24 @@
+name: license_review
```

Comment:
put comment on top pointing to the awesome README you wrote! if the job fails this might be the first place someone looks.